gke/main.tf

```diff
@@ -1,24 +1,24 @@
 resource "random_string" "cluster_name" {
-  length  = 18
+  length  = 8
   special = false
   upper   = false
   number  = false
 }
 
 resource "google_container_cluster" "gke-cluster" {
   name     = "cap-${random_string.cluster_name.result}"
-  location = "${var.location}"
+  location = var.location
 
   # We can't create a cluster with no node pool defined, but we want to only use
   # separately managed node pools. So we create the smallest possible default
   # node pool and immediately delete it.
   remove_default_node_pool = true
 
   initial_node_count = 1
-  min_master_version = "${var.k8s_version}"
-  node_version = "${var.k8s_version}"
+ # min_master_version = "${var.k8s_version}"
```

Comment:
is this needed anymore? if no we should remove it.

Reply:
Removed.